src/app/components/Metadata/index.test.jsx

```diff
@@ -74,28 +74,6 @@ describe('Metadata', () => {
     '@BBCNews',
   );
 
-  metadataSnapshotTest(
```

Comment:
Why has this snapshot test been removed?

Reply:
This was testing what happens when title is `null`, which it can no longer be :)